tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/embed_block.html

```diff
@@ -1 +1,33 @@
-<div class="grid__embed streamfield__embed">{{ value }}</div>
+{% load static %}
+
+<div class="grid__embed streamfield__embed youtube-video-container">
+    {% if is_youtube %}
+        <div class="youtube-video-placeholder">
+            {% if thumbnail_url %}
+                <img src="{{ thumbnail_url }}" alt="Video Thumbnail">
```

Comment:
I don't think we're likely to want to use that placeholder icon so maybe just leave it out of the pr and add a comment that we need a fallback placeholder image.